src/base/utils/misc.cpp

```diff
@@ -633,4 +637,156 @@ Path Utils::Misc::windowsSystemPath()
     }();
     return path;
 }
-#endif
+
+bool Utils::Misc::isTorrentFileAssocSet()
```

Comment:
Has this code been moved without changes?

Reply:
Only with very minor changes: adding `::` prefix before OS native API functions.